homepage/css/style.css

```diff
@@ -1,12 +1,45 @@
 /******************************************
 /* CSS
 /*******************************************/
+html{
+  font-family: 'Noto Sans Display', sans-serif;
+  font-size: 62.5%;
+}
 
 /* Box Model Hack */
 *{
   box-sizing: border-box;
 }
 
+h2{
+  font-weight: 300;
+  font-size: 2.5rem;
+}
+
+.header-title{
+  color: #b31b10;
+  margin-top: 3.9%;
+}
+header {
+  display: flex;
+}
+
+.logo{
+  width: 15%;
+  margin-right:-5.5%;
+}
+
+.logo-name{
+  flex-direction: row;
+  justify-content: flex-start;
+  display: flex;
+}
+
+li{
+  list-style: none;
+  font-size: 2rem;
+}
```

Comment:
We could add a selector for the `ul`  inside the header to have the lists sitting side by side.

![image](https://user-images.githubusercontent.com/38200521/168453188-d292e42d-7a32-426b-bda5-4d7a394f09db.png)

Reply:
We could also add some `hover` styles when the user hovers with the cursor on the link.

Reply:
Hi Ilba, 

I made the changes and I am going to push the code to my branch and issue another pull request. I still have no idea how to do the hamburger. I did not have time to look into that. 
<img width="1361" alt="Screen Shot 2022-05-15 at 4 44 42 PM" src="https://user-images.githubusercontent.com/89028640/168493083-d1e14aac-50f1-4dcd-a098-bb4f7bd9b118.png">